Update menu underboss

FranciscoBelo18 opened the pull request:
Feito o menu underboss com algumas alteraçoes no que ja estava lá feito sendo adicionado alguns passos , nos negocios feito a simplificação de codigo na parte do AtualizarValores sendo feito um metodo abstrato para isso, criei uma interface policiavel para ver quando a policia intervem nos negocios e tudo mais nos negocios plociaveis(foi posto o casino como policiavel)

FranciscoBelo18 committed:
Update menu underboss

Feito o menu underboss com algumas alteraçoes no que ja estava lá feito sendo adicionado alguns passos , nos negocios feito a simplificação de codigo na parte do AtualizarValores sendo feito um metodo abstrato para isso, criei uma interface policiavel para ver quando a policia intervem nos negocios e tudo mais nos negocios plociaveis(foi posto o casino como policiavel)

FranciscoBelo18 committed:
Corrigido erro

Passou este errozinho aqui apenas

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Main.java:
```diff
@@ -103,12 +103,22 @@ public static void main(String[] args) {
                                     System.out.println("Introduza o id do mafioso que deseja libertar da prisao");
                                     preso = scan.nextInt();
                                     prisao.libertarPreso(preso);
+                                    familia1.setRiqueza(familia1.getRiqueza()*0.05);//pagamento de libertar
```
Isto devia de ficar dentro do metodo libertarPreso

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Policiavel.java:
```diff
@@ -0,0 +1,14 @@
+
+/**
+ *
+ * @author franc
```
Apaga isto

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Policiavel.java:
```diff
@@ -0,0 +1,14 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
```
Apaga isto

FranciscoBelo18 committed:
Corrigido o libertarPreso

Reparei ao corrigir o tal comentario sobre a riqueza, que o libertarPreso estava ainda incompleto e ja fiz entao a correçao do seu codigo
